frontend/src/components/LandingPage.tsx

```diff
@@ -1,11 +1,19 @@
+import {useNavigate} from "react-router-dom";
+
 export default function LandingPage() {
 
+    const navigate = useNavigate()
+
+    const routeToTrueCategories = () => {
+        navigate("/categories")
+    }
+
     return (
         <div>
             <div>
                 Hier wird die App beschrieben und es erscheint ein Empfangstext
             </div>
-            <button>Start</button>
+            <button onClick={routeToTrueCategories}>Start</button>
```

Comment:
```suggestion
            <button onClick={() => {
        navigate("/categories")}>Start</button>
```

just to save a few lines :)